src/components/gateway/GatewayCreatePage.tsx

```
@@ -756,6 +783,8 @@ const GatewayCreatePage: React.FC<GatewayCreatePageProps> = ({ onFormChange }) =
     {
       name: t('Protocol'),
       nextButtonText: t('Next'),
+      isNextDisabled: !isProtocolStepValid,
```

ESLint

[error] Replace `······` with `········`

(prettier/prettier)